### ISSUE TITLE
2.8.3: Minimize error after upgrade

### ISSUE THREAD

**Issue by ggascoigne** [lingui/js-lingui#541]
After upgrading from 2.7.4 to 2.8.3 I get the following build error.

```
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/@lingui/loader!./node_modules/file-loader/dist/cjs.js??ref--6-oneOf-7!./src/locales/en/messages.po:1

Read more here: https://bit.ly/CRA-build-minify

error Command failed with exit code 1.
```

This is in a create-react-app based project.

I've got no real idea where to start debugging this one.

- jsLingui version ``lingui --version``
```
2.8.3
```

- Babel version ``npm list babel-core``
```
ui@0.1.0 /Users/ggp/dev/tw-git/devops-ui
└── babel-core@7.0.0-bridge.0
```
- Your Babel config (e.g. `.babelrc`) or framework you use (Create React App, Meteor, etc.)
create-react-app 3.0.1

**Comment by tricoder42** [comment]
Hi @ggascoigne, the advice mentioned on CRA pages doesn't help? https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify

**Comment by ggascoigne** [comment]
Which basically says use the latest react-scripts, which is what I'm doing.

**Comment by youngbob** [comment]
I tracked down what is happening here (same codebase as @ggascoigne, using create-react-app 3.2.0 ). We had set `compileNamespace` to `es` instead of the default `cjs`. On 2.7.4, the "lingui compile" command would generate compiled messages using "export default", but the webpack chunks created by @lingui/loader would be using "module.exports=". On 2.8.3, that behavior changed, and our webpack chunks started using "export default", which led to code like:

```javascript
(this["webpackJsonpui"] =this["webpackJsonpui"] || []).push([[23],{
 
/***/1311:
/***/ (function(module, exports) {
 
/* eslint-disable */export default{languageData:{"plurals"
```

which then threw a Terser error about export only being allowed at the top level.

I've solved the problem by changing compileNamespace back to cjs, but the behavior change here seems like a bug.
